Real Analysis/real-analysis-notes.tex

```diff
@@ -404,5 +404,174 @@ \subsection{Point-Set Topology}
   \]
 \end{proof}
 \subsection*{(1/26/2017) Lecture 4}
+\subsection{Measurable spaces}
+\begin{defn}
+  Given a set $X$ and a $\sigma$-algebra $\A$ on $X$, we call the pair
+  $(X,\A)$ a \emph{measurable set}.
+\end{defn}
+In the language of categories, we have objects and the morphisms
+between the objects. To put our discussion in context, here is a table
+of some categories with their objects and morphisms.
+\begin{center}
+  \begin{tabular}{|c|c|c|}
+    \hline && \\
+    Category name& Objects & Morphisms \\
+    \hline &&\\
+    Linear algebra & Vector spaces & Linear maps \\
+    Group theory & Groups & Group homomorphisms \\
+    Topology & Topological spaces $(X,\Top)$ & Continuous maps \\
+    Measurable sets & $(X,\A)$ & Measurable maps (\S 2.1 Folland) \\
+    \hline
+  \end{tabular}
+\end{center}
+This leads us to the question, what is a measurable map?
+\begin{defn}
+  Given $(X,\A)$ and $(Y,\mathcal{B})$ 2 measurable spaces, we say
+  that $f: X \to Y$ is an $(\A,\mathcal{B})$ measurable map if and
+  only if for all $z \in \mathcal{B}$, $f^{-1}(z) \in \A$. 
+\end{defn}
+With this definition, we can now provide another characterization of
+product $\sigma$-algebras like we did for product topologies.
+\begin{prop}
+  Let $X = \prod_{\alpha \in A} X_\alpha$ with $(X_\alpha,\M_\alpha)$
+  a family of measurable spaces. Then $\bigotimes_{\alpha \in A} \M_\alpha$ is the
+  smallest $\sigma$-algebra which makes all the natural maps
+  $\pi_\alpha: X \to X_\alpha$ measurable. 
+\end{prop}
+At this point, we provide note that we are headed in the direction of
+an idea called a measure space $(X,\A,\mu)$ where $(X,\A)$ is a
+measurable space and $\mu$ is a ``measure'' defined on sets in
+$\A$. Given a $f: X \to \R$ that is measurable, we will be able to
+define $\int f \in \R$. The most important examples of this are $X =
+\R, \R_n, \Q_p, \Q_p^n$. However, in probability theory and quantum
+field theory, $X = \R^\R$ (the set of function from $\R$ to $\R$) is
+also incredibly important.
+\begin{example}
+  Let $A = X$ and for all $\alpha \in A$, let $Y_\alpha = Y$. Then
+  $Y^X = \prod_{\alpha \in A} Y_\alpha$. This means that we can think
+  of a $y \in Y^X$ as $y = (y_\alpha)_{\alpha \in A}$ where $y: A \to
+  \bigcup_{\alpha \in A}$ such that for all $\alpha \in A$, $y(\alpha)
+  = y_\alpha \in Y_\alpha$. 
```

Comment:
I agree we should be consistent and use just $X$ after the initial sentence. An uncountable product looks like a function.